### PR TITLE
Various updates to the MediaService doc

### PR DESCRIPTION
**Pull request by Jeavon**
Following https://github.com/umbraco/UmbracoDocs/issues/4440 I have made a few updates to the Media Service doc and updated the code samples from the Samples Project.

### CODE REVIEW

File: Reference/Management/Services/MediaService/Index.md [hunk 1]
```diff
@@ -92,24 +91,31 @@ In addition to the three mandatory parameters as shown above, you may also speci
 
 You can specify a `Stream` for the contents of the file that should be created.
 
-As an example, if you have a file on disk, you can open a new stream for a file on the disk, and then create a new media for that file in Umbraco:
+As an example, if you have a image on disk named `unicorn.jpg` in the images folder of `wwwroot`. You can open a new stream for a file on the disk, and then create a new media item for that file in Umbraco:
```
Comment (sofietoft) [inline Reference/Management/Services/MediaService/Index.md:94]:
```suggestion
As an example, if you have an image on disk named `unicorn.jpg` in the images folder of `wwwroot`. You can open a new stream for a file on the disk, and then create a new media item for that file in Umbraco:
```

**Review by sofietoft — COMMENTED** [review]
Looks great @Jeavon ! Just a tiny little suggestion, then we're all good to go!

**Comment by Jeavon** [comment]
Accepted, thanks @sofietoft

**Comment by sofietoft** [comment]
Great 👋 
I'll merge it right in!

**Comment by biapar** [comment]
Will I mentioned about this fix?

**Comment by sofietoft** [comment]
Hi @biapar ! 👋 
I'm not entirely sure I understand your request here - could elaborate a bit on what you mean?

**Comment by biapar** [comment]
I mean if my Github alias will be listed in the contribution list.

**Comment by sofietoft** [comment]
Ah, I see! Thanks for explaining.

I'm afraid the Issue you reported does not count as a contribution to this year's Hacktoberfest.
It does not quite fit the official guidelines we follow: https://hacktoberfest.com/about/#low-or-non-code

There are, however, still a lot of tasks you can pick up for this years Hacktoberfest.
Have a look at the Issue we have created: https://github.com/umbraco/UmbracoDocs/issues/4416
There is still a lot of improvements that can be done when it comes to readability of our docs 💪

**Pull request merged**